### PR TITLE
feat: workflows in scala protobuf sdk

### PR DESCRIPTION
**Pull request by aludwiko**

### CODE REVIEW

**Review by octonato — APPROVED** [review]
LGTM

Left a few minor typo fixes.

File: samples/scala-protobuf-transfer-workflow-compensation/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Implementing a Counter as a Value Entity
```
[the remark this answers is not in the document]
Comment (aludwiko) [reply samples/scala-protobuf-transfer-workflow-compensation/README.md:1]: fixed

File: samples/scala-protobuf-transfer-workflow-compensation/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# If you're looking to use eventing with Google PubSub, to get an emulator running:
+# - add property "-Dkalix.proxy.eventing.support=google-pubsub-emulator" to the JAVA_TOOL_OPTIONS environment map under the kalix-runtime service
+# - uncomment the env var PUBSUB_EMULATOR_HOST and the section below for gcloud-pubsub-emulator service
+version: "3"
+services:
+  kalix-runtime:
+    image: gcr.io/kalix-public/kalix-runtime:1.1.29-20-9ecb0999
```
[the remark this answers is not in the document]
Comment (aludwiko) [reply samples/scala-protobuf-transfer-workflow-compensation/docker-compose.yml:7]: done

File: samples/scala-protobuf-transfer-workflow-compensation/src/main/scala/com/example/transfer/api/TransferWorkflow.scala [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+    val withdraw = step("withdraw")
+      .asyncCall((withdraw: Withdraw) => {
+        logger.info("Running withdraw step: " + withdraw)
+        timers
+          .cancel("acceptationTimout-" + currentState().transferId)
+          .flatMap(_ => {
+            val withdrawRequest = WithdrawRequest(withdraw.from, withdraw.amount)
+            components.walletEntity.withdraw(withdrawRequest).execute()
+          })
+      })
```
[the remark this answers is not in the document]
Comment (aludwiko) [reply samples/scala-protobuf-transfer-workflow-compensation/src/main/scala/com/example/transfer/api/TransferWorkflow.scala:51]: good point, fixing it

File: samples/scala-protobuf-transfer-workflow/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# Implementing a Counter as a Value Entity
```
[the remark this answers is not in the document]
Comment (aludwiko) [reply samples/scala-protobuf-transfer-workflow/README.md:1]: fixed

**Pull request merged**